ansible_collections/arista/avd/roles/eos_designs/python_modules/connected_endpoints/port_channel_interfaces.py

```diff
@@ -174,3 +176,32 @@ def _get_port_channel_subinterface_cfg(self, subinterface: dict, adapter: dict,
             }
 
         return strip_null_from_data(port_channel_interface)
+
+    def _check_duplicate(self, candidate_port_channel_config, port_channel_interfaces) -> None:
+        """
+        This function assumes that port_channel_interfaces list DO NOT contain duplicate port-channel names
+
+        This check function does two things:
+        1. Check if the port_channel_config["name"] is alreadt present in port_channel_interfaces computed so far
```

Comment:
```suggestion
        1. Check if the port_channel_config["name"] is already present in port_channel_interfaces computed so far
```

Reply:
Addressed in [28984ed](https://github.com/aristanetworks/ansible-avd/pull/2651/commits/28984ed3cabd86a56432925167173adbe8056138)